docs/user_guide/data.rst

```diff
@@ -88,4 +88,8 @@ The parameter "``same_time_dims_as``" takes a dictionary with axes indices or ax
 as key and the `DimensionTag` as value.
 For the above example, there is no difference in using `'T'` or `1` as key.
 
+In case you want to synchronize layer outputs, a "``reinterpret_data``"
```

Comment:
This is a bit misleading, or at least unclear. What does it mean to "synchronize layer outputs"? Probably you mean to say to merge spatial dimension tags? Otherwise I'm also not really sure what you mean here.

Reply:
I will update this formulation